Block/Widget/Posts.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Created by PhpStorm.
```

Comment:
I think it's a lot nicer to remove this comment tag and add a phpdoc right before the class name